### PR TITLE
[docs] use Terminal on more pages, tweak cmdCopy

### PR DESCRIPTION
**Pull request by Simek**
# Why & how

This PR focuses on adding Terminal component on various pages, which currently were using `bash` code blocks. I have only updated the block which seems to have a sense, but there might be few more places where we can use the component.

Additionally, in this changeset I have removed no longer necessary manually added `cmdCopy` for Terminals with only one valid command line. Also I have tweak the Terminal appearance inside lists (`li` tags), because old list style was overwriting the `display` of Terminal wrapper.

# Test Plan

The changes were tested by running docs website locally.

# Checklist

- [X] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/versions/unversioned/sdk/in-app-purchases.md [hunk 1]
```diff
@@ -16,16 +16,10 @@ import PlatformsSection from '~/components/plugins/PlatformsSection';
 
 <APIInstallSection hideBareInstructions />
 
-This module is currently only available in the [bare](../../../introduction/managed-vs-bare.md#bare-workflow) workflow.
+This module is currently only available in the [bare](/introduction/managed-vs-bare.md#bare-workflow) workflow.
 
 You must ensure that you have [installed and configured Expo modules](/bare/installing-expo-modules.md) before continuing.
 
-### Add the package to your dependencies
```
Comment (FiberJW) [inline docs/pages/versions/unversioned/sdk/in-app-purchases.md:23]: Is this missing the `<Terminal />` replacement?

File: docs/pages/versions/v42.0.0/sdk/in-app-purchases.md [hunk 2]
```diff
@@ -11,15 +11,11 @@ import PlatformsSection from '~/components/plugins/PlatformsSection';
...
+<APIInstallSection hideBareInstructions />
 
-You must ensure that you have [installed and configured the `react-native-unimodules` package](https://github.com/expo/expo/tree/main/packages/react-native-unimodules) before continuing.
-
-### Add the package to your dependencies
+This module is currently only available in the [bare](/introduction/managed-vs-bare.md#bare-workflow) workflow.
 
-```
-npm install expo-in-app-purchases
-```
```
Comment (FiberJW) [inline docs/pages/versions/v42.0.0/sdk/in-app-purchases.md:22]: Same here about missing command section

File: docs/pages/versions/v43.0.0/sdk/in-app-purchases.md [hunk 3]
```diff
@@ -12,15 +12,11 @@ import PlatformsSection from '~/components/plugins/PlatformsSection';
...
+<APIInstallSection hideBareInstructions />
 
-You must ensure that you have [installed and configured Expo modules](/bare/installing-expo-modules.md) before continuing.
-
-### Add the package to your dependencies
+This module is currently only available in the [bare](/introduction/managed-vs-bare.md#bare-workflow) workflow.
 
-```
-npm install expo-in-app-purchases
-```
```
Comment (FiberJW) [inline docs/pages/versions/v43.0.0/sdk/in-app-purchases.md:23]: +1

File: docs/pages/versions/v44.0.0/sdk/in-app-purchases.md [hunk 4]
```diff
@@ -12,15 +12,11 @@ import PlatformsSection from '~/components/plugins/PlatformsSection';
...
-This module is currently only available in the [bare](../../../introduction/managed-vs-bare.md#bare-workflow) workflow.
+<APIInstallSection hideBareInstructions />
 
-You must ensure that you have [installed and configured Expo modules](/bare/installing-expo-modules.md) before continuing.
-
-### Add the package to your dependencies
+This module is currently only available in the [bare](/introduction/managed-vs-bare.md#bare-workflow) workflow.
 
-```
-npm install expo-in-app-purchases
```
Comment (FiberJW) [inline docs/pages/versions/v44.0.0/sdk/in-app-purchases.md:22]: same

File: docs/pages/versions/v45.0.0/sdk/in-app-purchases.md [hunk 5]
```diff
@@ -16,16 +16,10 @@ import PlatformsSection from '~/components/plugins/PlatformsSection';
...
 
-This module is currently only available in the [bare](../../../introduction/managed-vs-bare.md#bare-workflow) workflow.
+This module is currently only available in the [bare](/introduction/managed-vs-bare.md#bare-workflow) workflow.
 
 You must ensure that you have [installed and configured Expo modules](/bare/installing-expo-modules.md) before continuing.
 
-### Add the package to your dependencies
-
-```
-npm install expo-in-app-purchases
```
Comment (FiberJW) [inline docs/pages/versions/v45.0.0/sdk/in-app-purchases.md:26]: same

**Review by FiberJW — APPROVED** [review]
If the removal of the IAP install command was intentional, LGTM. If not, fix before merging

**Comment by Simek** [comment]
@FiberJW [`APIInstallSection` component](https://github.com/expo/expo/blob/main/docs/components/plugins/InstallSection.tsx) is used on those pages, which includes the install command within terminal. Those removed parts were basically duplicates, remains of the content before `APIInstallSection` existed.

File: docs/pages/distribution/optimizing-updates.md [hunk 6]
```diff
@@ -32,10 +34,10 @@ Many images can be reduced by more than 30% in size if they haven't been previou
 
 Another way to optimize images is to re-encode them using an optimizer like [expo-optimize](https://github.com/expo/expo-cli/tree/main/packages/expo-optimize#-welcome-to-expo-optimize), which optimizes all compatible images in you Expo project:
 
-```
-npm install -g sharp-cli
-npx expo-optimize <project-directory> [options]
-```
+<Terminal cmd={[
+  '$ npm install -g sharp-cli',
```
[the remark this answers is not in the document]
Comment (Simek) [reply docs/pages/distribution/optimizing-updates.md:38]: For multiline command we do not generate copy automatically. In this case I have missed adding `cmdCopy` manually, nice catch!

**Comment by Simek** [comment]
Thanks to you suggestion I have spotted that we were using wrong install command for the bare only package, that's has been fixed now. 🙂

File: docs/pages/distribution/optimizing-updates.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (FiberJW) [reply docs/pages/distribution/optimizing-updates.md:38]: Yeah i was also thinking that hiding the copy button would be appropriate here. Or we can add a cmdCopy override with a chained command `npm i -q sharp-cli && npx expo-optimize ...`

[the remark this answers is not in the document]
Comment (Simek) [reply docs/pages/distribution/optimizing-updates.md:38]: Let's leave the block without the copy button for now. User still can copy the whole Terminal content or just one line, and I feel like adding chained command with the placeholders or with missing part (`expo-optimize` case) don't add to much. Also, usually we skip copy command from terminals with placeholder or example values.

**Pull request merged**